demos/interpreter/step-execution.html

```diff
@@ -153,21 +153,20 @@ <h1><a href="https://developers.google.com/blockly/">Blockly</a> &gt;
       // Add an API function for the alert() block, generated for "text_print" blocks.
       interpreter.setProperty(scope, 'alert',
           interpreter.createNativeFunction(function(text) {
-        text = text ? text.toString() : '';
+        text = arguments.length ? text : '';
```

Comment:
Out of curiosity why not `text != null``?

Reply:
```
alert(null) => 'null'
alert(undefined) => 'undefined'
alert() => ''
```
I need to print a stringified version of every value -- except if there is no argument.  If `text === undefined` that's not enough to know what to print.

Reply:
Ah I see. Makes sense.